frontend/src/Explorer/View/Address.purs

```diff
@@ -90,18 +90,27 @@ addressDetailRowItems (CAddressSummary address) lang =
       , label: translate (I18nL.common <<< I18nL.cAddress) lang
       , value: address ^. (caAddress <<< _CAddress)
       , mCurrency: Nothing
-    }
+      }
     , { id: "1"
       , label: translate (I18nL.common <<< I18nL.cTransactions) lang
       , value: show $ address ^. caTxNum
       , mCurrency: Nothing
-    }
+      }
     , { id: "2"
```

Comment:
Ah, I see. And yes, it might be misleading to name it `id`. Because it is just a key attribute, which is needed by `React` https://facebook.github.io/react/docs/lists-and-keys.html#keys to avoid warnings about missing keys of children. `CAddressSummary` does not have an identifier or similar, which we could use to set value of a key. So we have to do it "by hand" setting "1", "2" etc.  I will change it from `id` to `key`.